src/components/Person.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${name}`}
+    </h2>
+
+    {age && (<p className="Person__age">{`I am ${age}`}</p>)}
+
+    <p className="Person__partner">
+      {
+        (isMarried)
+          ? `My ${(sex === 'm')
+            ? "wife's"
+            : "husband's"} name is ${partnerName}`
+          : `I am not married`
+      }
+    </p>
+  </section>
+);
+
+Person.propTypes = {
+  name: PropTypes.string.isRequired,
+  age: PropTypes.number.isRequired,
+  sex: PropTypes.string.isRequired,
+  isMarried: PropTypes.string.isRequired,
+  partnerName: PropTypes.string.isRequired,
+};
```

Comment:
I don't know how its works. Age and partneName should be not required. Olya hasn't age and Alex hasn't partnerName